SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -480,6 +480,22 @@ bool USpatialClassInfoManager::IsSublevelComponent(Worker_ComponentId ComponentI
 	return SchemaDatabase->LevelComponentIds.Contains(ComponentId);
 }
 
+TArray<Worker_ComponentId> USpatialClassInfoManager::GetComponentIdsForComponentType(const ESchemaComponentType ComponentType)
+{
+	switch (ComponentType)
+	{
+	case ESchemaComponentType::SCHEMA_Data:
+		return SchemaDatabase->DataComponentIds;
+	case ESchemaComponentType::SCHEMA_OwnerOnly:
+		return SchemaDatabase->OwnerOnlyComponentIds;
+	case ESchemaComponentType::SCHEMA_Handover:
+		return SchemaDatabase->HandoverComponentIds;
+	default:
+		UE_LOG(LogSpatialClassInfoManager, Warning, TEXT("Component type %d not recognised."), ComponentType);
```

Comment:
This probably should be a `checkNoEntry()`? That case should never hit.

Reply:
This is just future proofing, if more component types are added and this is not updated this warning will be triggered. Any case of hitting this is a programming error (maybe it should error and exit instead?), it's just here to be explicit

Reply:
oh that is what checknoentry does

Reply:
sweet

Reply:
changed to an error and added check no entry